framework_tvb/tvb/adapters/exporters/tvb_linked_export.py

```diff
@@ -74,24 +74,25 @@ def export_data_with_references(self, export_data_zip_path, data_export_folder):
 
         return None, export_data_zip_path, True
 
```

Comment:
Could the export method from here compute the filename first and then just make one call to the storage giving the list of datatypes to be exported and the filename for the ZIP?
If the separate methods to copy the dts and then archive them can be put into a single method, could that be reused in other cases? Eg: I see write_zip_folder_with_links and export_datatypes methods in the StorageInterface which seem to be peforming a similar action.